plugins/test/unit/plugins/importers/yum/test_sync.py

```diff
@@ -1,7 +1,11 @@
 from copy import deepcopy
 from cStringIO import StringIO
 import os
-import unittest
+
+try:
+    import unittest2 as unittest
+except ImportError:
```

Comment:
That makes sense. If test code uses something that was introduced in python 2.7, won't that fail on environments like EL6 which won't install unittest2?